.github/workflows/run_cell-type-ewings.yml

```
@@ -34,36 +35,15 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
-      - name: Download test data
-        run: ./download-data.py --test-data --format "sce,anndata" --samples SCPCS000490
-
-      - name: Set up R
-        uses: r-lib/actions/setup-r@v2
-        with:
-          r-version: 4.4.0
-          use-public-rspm: true
-
-      - name: Set up pandoc
-        uses: r-lib/actions/setup-pandoc@v2
-
-      - name: Install additional system dependencies
+      - name: Install awscli
         run: |
-          sudo apt-get install -y libcurl4-openssl-dev libglpk40 jags
+          pip install awscli
```

This gets an old version, I'm pretty sure, so we should probably get it the way AWS recommends:
```suggestion
          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
          unzip awscliv2.zip
          sudo ./aws/install
```